src/Console/Command/FixCommand.php

```diff
@@ -254,6 +255,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (null !== $stdErr) {
             $stdErr->writeln(Application::getAboutWithRuntime(true));
 
+            // @TODO remove when parallel runner is mature enough and works as expected
```

Comment:
btw, how we want to determine that ?

Reply:
There's no easy heuristic for that. My idea was to release this feature and observe the backlog for some time, and of course use it for development. Also try to collect feedback from several sources (Twitter, Symfony Slack, Roave Discord, Mastodon etc). And then just remove experimental flag if there are not many issues with it.